services/udp/README.md

```diff
@@ -1,13 +1,125 @@
-# Configuration
+# The UDP Input
+
+## A note on UDP/IP OS Buffer sizes
+
+Some OSes (most notably, Linux) place very restricive limits on the performance
+of UDP protocols. It is _highly_ recommended that you increase these OS limits to
+at least 8MB before trying to run large amounts of UDP traffic to your instance.
+8MB is just a recommendation, and should be adjusted to be inline with your
+`read-buffer` plugin setting.
+
+### Linux
+Check the current UDP/IP receive buffer limit by typing the following commands:
+
+```
+sysctl net.core.rmem_max
+```
+
+If the values are less than 8388608 bytes you should add the following lines to the /etc/sysctl.conf file:
+
+```
+net.core.rmem_max=8388608
+```
+
+Changes to /etc/sysctl.conf do not take effect until reboot.  To update the values immediately, type the following commands as root:
+
+```
+sysctl -w net.core.rmem_max=8388608
+```
+
+### BSD/Darwin
+
+On BSD/Darwin systems you need to add about a 15% padding to the kernel limit
```

Comment:
@otoolep finally tracked down the BSD/Darwin failures I was seeing to this. This PR should be good to merge now.